book/src/gobject_memory_management.md

```diff
@@ -111,7 +108,7 @@ We can improve that by using the `glib::clone!` macro.
 {{#rustdoc_include ../listings/gobject_memory_management/2/main.rs:callback}}
 ```
 
-Just like `Rc<RefCell<T>>`, GObjects are reference-counted and mutable.
+Just like `Rc<Cell<T>>`, GObjects are reference-counted and mutable.
```

Comment:
This should stay as RefCell, most of the times you don't have a Copy type

Reply:
Sure, there was obvious a miscommunication in our last discussion.
I will then revert everything back to RefCell, because I don't want to use Cell AND RefCell without need.

Reply:
Well, it's not a need, it's about "correctness". The book should use RefCell where it fits & Cell where it does.

Reply:
Cell is correct in all instances I've used it. So is RefCell.

However, RefCell is useful in more cases.

Reply:
@sdroege any opinions on:
1) everything RefCell
2) Cell where "it makes sense"?

If we choose 2, we definitely need a discussion where it makes senses, and how to explain/motivate it to the reader.

Reply:
Use `Cell` where you can, use `RefCell` elsewhere, and where needed use `Mutex`, etc. Like you'd do in a proper application. You use the tool that fits your usage best.